### ISSUE TITLE
Windows with style "PositionPlacement Center" split between monitors

### ISSUE THREAD

**Issue by NsCDE** [fvwmorg/fvwm3#648]
As title said. "Style appname PositionPlacement Center" does not work right anymore.

fvwm3 1.0.5 (1.0.4-115-g6b5f3661)
with support for:  ReadLine, XPM, PNG, SVG, Shape, XShm, SM, Bidi text, XRandR, XRender, XCursor, XFT, NLS

## Expected Behaviour

Placed in the center of the current monitor which holds the mouse pointer.

## Actual Behaviour

Placed in the "real" center - between two monitors.

See pic example attached.
![ScreenAreaShot WSP_Four 20211216 171202](https://user-images.githubusercontent.com/48650374/146408064-8c64c148-5337-4f8a-b44d-7c15a4cb09d1.png)

**Comment by NsCDE** [comment]
Steps to reproduce with plain default FVWM setup:

1. Copy <prefix>/share/fvwm3/default-config as ~/.fvwm
2. Edit ~/.fvwm/config, add `Module FvwmMFL` into StartFunction
3. Login to this session / start fvwm with this config
4. In one terminal window, enable the second screen and restart fvwm:

```
xrandr --output Virtual-1 --mode 1400x1050 --primary --output Virtual-2 --mode 1400x1050 --right-of Virtual-1
FvwmCommand Restart
```
Replace RandR names of the monitors above with you actual names.

Then, configure xterm to have Style PositionPlacement Center and start it:

```
FvwmCommand 'Style XTerm PositionPlacement Center'
xterm &
```

Xterm will appear between two monitors, in the *real* center instead of center of the current monitor.

The same behaviour can be observer while calling Quit from the menu: Quit form will appear between two monitors.
Latest release behaves well BTW.

**Comment by NsCDE** [comment]
This bug has been fixed by Dominik Vogt: 44e21c5bc2460fd4ae625a808e9e57dc416a65d9 in the master

Quote:
-------------------
> Yes, now it works. So you replaced $[w.screen] with ... something as I see
> in git diff. Is it [c]urrent or is it aware if it should start on exact
> screen?

It uses the window's screen which has already been set by the time
the window is placed.
-------------------
